### ISSUE TITLE
new features 

### ISSUE THREAD

**Issue by GayanEranga92** [expressjs/generator#216]
add mocha chai unit testing support (unit testing folders and config files)

**Comment by dougwilson** [comment]
Yes, sounds like a great idea. It can be added as a flag for users to choose the framework (for when the next person wants jest for example). Like maybe `--test=mocha` or something.

**Comment by oviecodes** [comment]
Wonderful idea, had the same thought for passport authentication. @dougwilson  Maybe I could take this up once I'm done learning about testing. Could add support for mocha and jest framework
